staging/src/k8s.io/apiserver/pkg/util/feature/feature_gate.go

```diff
@@ -31,3 +31,13 @@ var (
 	// Top-level commands/options setup that needs to modify this feature gate should use DefaultMutableFeatureGate.
 	DefaultFeatureGate featuregate.FeatureGate = DefaultMutableFeatureGate
 )
+
+// Enabled returns true if the key is enabled in the default FeatureGate.
+func Enabled(key featuregate.Feature) bool {
+	return DefaultFeatureGate.Enabled(key)
+}
+
+// KnownFeatures returns a slice of strings describing the default FeatureGate's known features.
+func KnownFeatures() []string {
+	return DefaultFeatureGate.KnownFeatures()
+}
```

Comment:
Nice, I like the simplification.

Reply:
It looks like you missed utilfeature->feature rename in `authorizer/rbac/bootstrappolicy/controller_policy.go`

Reply:
That commit didn't convert ALL users, just users that were converted to the package-funcs.  Subsequent commits should get the rest.